var/spack/repos/builtin/packages/apple-glu/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class AppleGlu(Package):
+    """Shim package for Apple implementation of OpenGL Utility Libray (GLU)"""
+
+    homepage = ""
+
+    maintainers = ["aphecetche"]
+
+    has_code = False
+
+    version("1.3.0")
+
+    provides("glu@1.3", when=sys.platform == "darwin")
```

Comment:
As mentioned above, this is not needed as the `conflicts()` address the support issue.